pkg/functionconfig/types.go

```diff
@@ -66,11 +66,14 @@ type Volume struct {
 
 // Trigger holds configuration for a trigger
 type Trigger struct {
-	Class                                 string            `json:"class"`
-	Kind                                  string            `json:"kind"`
-	Name                                  string            `json:"name"`
-	Disabled                              bool              `json:"disabled,omitempty"`
+	Class    string `json:"class"`
+	Kind     string `json:"kind"`
+	Name     string `json:"name"`
+	Disabled bool   `json:"disabled,omitempty"`
```

Comment:
Why changing formatting here?

Reply:
I think it was auto-formatted by gofmt because of the comment below.
I can move `MaxWorkers` down to keep a prettier formatting.

Reply:
@TomerShor then nvm, let's leave as is

Reply:
I actually prefer moving the deprecated fields to the bottom, to make it clearer it shouldn't be used